distributed/protocol.py

```diff
@@ -63,6 +63,14 @@
     default_compression = 'lz4'
 
 
+with ignoring(ImportError):
+    import blosc
+    blosc.set_nthreads(2)
```

Comment:
I found that benefits of compression fall off a bit after the first few cores.  I'm also concerned about cores fighting for resources.  Often our cores are busy doing actual work.